appoetry1/AppDelegate.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  AppDelegate.swift
+//  appoetry1
+//
+//  Created by Kristaps Brēmers on 25.02.19.
+//  Copyright © 2019. g. Chili. All rights reserved.
+//
+
+import UIKit
+import Firebase
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+    
+    var window: UIWindow?
```

Comment:
Mr Ed has some new approach to share with you. That thing will resolve ?? (double option) in FlowController protocol.